integration-test/src/test/java/com/datastrato/gravitino/integration/test/spark/iceberg/SparkIcebergCatalogIT.java

```diff
@@ -559,4 +632,32 @@ static IcebergTableWriteProperties of(
       return new IcebergTableWriteProperties(isPartitionedTable, formatVersion, writeMode);
     }
   }
+
+  private SparkIcebergTable getSparkIcebergTableInstance(String tableName)
```

Comment:
could we get `snapshotId` or `SnapshotTimestamp` from SparkSQL not internal API?

Reply:
we have not support metadata tables yet, so we can not get `snapshotId` or `SnapshotTimestamp` from SparkSQL.

Reply:
i think we can refactor it when we support metadata tables.

Reply:
got it